components/dropdown/script.js

```diff
@@ -174,7 +174,10 @@ class GamefaceDropdown extends CustomElementValidator {
     */
     set selected(option) {
         // reset
-        if (option === null) return this.resetSelection();
+        if (option === null) {
+            this.resetSelection();
+            return;
+        }
```

Comment:
Why we changed this on multiple lines? What the linter is telling you here? IMO the previous change was better.

Reply:
The linter says that the setter can not return :D

Reply:
I also agree that the previous variant was better...the error isn't even true because `this.resetSelection();` returns undefines so this line `return this.resetSelection;` actually returns nothing...